modules/c3-assisted-installer-preparing-bastion-server.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-preparing-bastian-server_{context}"]
+= Preparing the {oci} Bastion server
```

Comment:
Does "Bastion" need to be capitalized? It looks like it isn't capitalized later in this content.